docs/source/gcm/user_guide/introduction.rst

```diff
@@ -10,6 +10,17 @@ The ``dowhy.gcm`` package provides a variety of ways to answer causal questions
 section :doc:`answering_causal_questions/index`. However, before diving into them, let's understand
 the basic building blocks and usage patterns it is built upon.
 
+If you find the GCM-based inference useful for your work, please cite us as follows:
```

Comment:
Suggestion: "GCM-based inference" -> "gcm module"

Reply:
Changed it to "package" instead of "module".